nixos/modules/image/amend-repart-definitions.py

```diff
@@ -53,7 +53,7 @@ def add_closure_to_definition(
 
             source = Path(line.strip())
             target = str(source.relative_to("/nix/store/"))
-            target = f":{target}" if strip_nix_store_prefix else ""
+            target = f":/{target}" if strip_nix_store_prefix else ""
```

Comment:
Are you sure this is correct? Maybe we can add a test that actually does the stripping.

Reply:
Without this I get:

```
 > /build/amended-repart.d/nixStore.conf:516: CopyFiles= path is not absolute, ignoring: zc9ryh74m3cyzzlkkm0pxsx0jq8jiqzl-tzdata-2023c
       > /build/amended-repart.d/nixStore.conf:517: CopyFiles= path is not absolute, ignoring: zg1dmivn1fhc0n7v3ra9yr6r0d32604s-X-Restart-Triggers
       > /build/amended-repart.d/nixStore.conf:518: CopyFiles= path is not absolute, ignoring: ziyf74lw0im3cpm8j7g9d0rswjk3y1nr-npth-1.6
       > /build/amended-repart.d/nixStore.conf:519: CopyFiles= path is not absolute, ignoring: zj1ah6hzvgw18fnp1xkx5w1wwjxgi7if-aws-sdk-cpp-1.11.118
       > /build/amended-repart.d/nixStore.conf:520: CopyFiles= path is not absolute, ignoring: zl4hzvv5rp9n0hr0xgf0kxl5vzwyzp33-libcap-2.69
       > /build/amended-repart.d/nixStore.conf:521: CopyFiles= path is not absolute, ignoring: zm9jq42h6kqfbxb9mh3gk2p1x4x4s60i-mdadm-4.2
       > /build/amended-repart.d/nixStore.conf:522: CopyFiles= path is not absolute, ignoring: zmxrj3pgfqc38jwycgdyaqs4nggm4pxd-libressl-3.7.3-man
       > /build/amended-repart.d/nixStore.conf:523: CopyFiles= path is not absolute, ignoring: zphbsjgw5xw9lwkgl1rvh8p79jd3z530-fc-53-no-bitmaps.conf
       > /build/amended-repart.d/nixStore.conf:524: CopyFiles= path is not absolute, ignoring: zr6pvi1b1cn5skj31wy7hi3yvfz469i9-unit-systemd-network-wait-online-.service
```